model-optimizer/extensions/middle/FakeSplitOutputs.py

```diff
@@ -23,10 +23,10 @@ def run_after(self):
 
     def find_and_replace_pattern(self, graph: Graph):
         for split_node in graph.get_op_nodes(op='Split'):
-            AddFakeOutputsToSplit.split_normalize_outputs(split_node)
+            AddFakeOutputsToSplit.node_normalize_outputs(split_node)
 
     @staticmethod
-    def split_normalize_outputs(node: Node):
+    def node_normalize_outputs(node: Node):
```

Comment:
please move this function to utils since you try to use it for different operations and make description for it

Reply:
Done.